src/finitediff.jl

```diff
@@ -14,266 +14,41 @@ end
     eps_cbrt * max(one(T), abs(x))
 end
 
-@inline function compute_epsilon_factor(fdtype::DataType, ::Type{T}) where T<:Real
+@inline function compute_epsilon_factor(fdtype::DataType, ::Type{T}) where T<:Number
     if fdtype==Val{:forward}
         return sqrt(eps(T))
     elseif fdtype==Val{:central}
         return cbrt(eps(T))
     else
         error("Unrecognized fdtype: must be Val{:forward} or Val{:central}.")
     end
+    nothing
```

Comment:
why nothing here? Shouldn't this error?

Reply:
The `nothing` is redundant there, it will never do anything, but it won't prevent the error. I probably just wrote it by reflex.